st2common/st2common/util/loader.py

```diff
@@ -202,6 +203,21 @@ def register_query_module(module_name):
     return QUERIER_MODULES_CACHE[module_name]
 
 
+def register_callback_module(module_name):
+    base_path = cfg.CONF.system.base_path
+    module_path = os.path.join(
+        "%s/runners/%s/callback/%s.py" % (base_path, module_name, module_name)
```

Comment:
This will actually combine everything as a string and not use `os.path.join` since you are passing a single argument (string) to `os.path.join`.

Should be:

```python
os.path.join(base_path, "runners", module_name, "callbacks", module_name + '.py')
```